src/Question.js

```diff
@@ -10,19 +10,32 @@ class Question extends Component {
       option: PropTypes.string.isRequired,
       correct: PropTypes.bool.isRequired
     })),
-    onClick: PropTypes.func.isRequired
+    onClick: PropTypes.func.isRequired,
+    questionNumber: PropTypes.number.isRequired,
+    userHasAnswered: PropTypes.bool.isRequired
+  }
+  getQuestionClassName() {
+    const questionClassName = {
+      correct ? 'answered' : 'wrong-answer'
```

Comment:
This incarnation was just the most recent thing that I had since you told me to push without discussing the questions I was asking